### ISSUE TITLE
Returned result drops select attributes when mixing strings and alias/arrays

### ISSUE THREAD

**Issue by getglad** [sequelize/sequelize#6772]
### **TLDR:**

If I mix strings and alias arrays in the `attributes` property, the returned result drops any string attributes. The query is written correctly, so I think this may be a problem with how Sequelize processes the result. If I make everything an alias, it runs as expected. SO, this is a minor bug (or maybe an intentional design choice), but mixing patterns would be nice for when I don't want to alias everything.
## What you are doing?

``` js
statsObject.push('tablename.otherColumn')
statsObject.push([sequelize.fn('MIN', sequelize.col('tablename.column')), 'column MIN'])
statsObject.push([sequelize.fn('MAX', sequelize.col('tablename.column')), 'column MAX'])
statsObject.push([sequelize.fn('AVG', sequelize.col('tablename.column')), 'column AVG'])
...
model.findAll({
  attributes: statsObject,
  ...
```
## What do you expect to happen?

The `dataValues` object in the result set to include `otherColumn`, which was passed as a string, in addition to the aggregate functions.
## What is actually happening?

`dataValues` includes all the aggregate columns, but does not include the `otherColumn` value.

```
result_set[0].dataValues {
  column AVG: 2.5330857835707348,
  column MAX: 4.8,
  column MIN: 1.1,
}
```

The query itself does include it:

```
SELECT
  [tablename].[otherColumn],
  MIN([tablename].[column]) AS [column MIN],
  MAX([tablename].[column]) AS [column MAX],
  AVG([tablename].[column]) AS [column AVG],
  ...
```

If I just copy and paste the query into SSMS, it returns the non-aliased column in the result, so I think this is Sequelize doing something wrong in processing/returning the result.
## Work Around

If I alias the column I want...

``` js
statsObject.push(['tablename.otherColumn', 'tablename.otherColumn'])
statsObject.push([sequelize.fn('MIN', sequelize.col('tablename.column')), 'column MIN'])
statsObject.push([sequelize.fn('MAX', sequelize.col('tablename.column')), 'column MAX'])
statsObject.push([sequelize.fn('AVG', sequelize.col('tablename.column')), 'column AVG'])
...
model.findAll({
  attributes: statsObject,
  ...
```

The query adds the alias...

```
SELECT
  [tablename].[otherColumn] as [otherColumn],
  MIN([tablename].[column]) AS [column MIN],
  MAX([tablename].[column]) AS [column MAX],
  AVG([tablename].[column]) AS [column AVG],
  ...
```

And it is in the result set:

```
result_set[0].dataValues {
  otherColumn: "hey I'm a result",
  column AVG: 2.5330857835707348,
  column MAX: 4.8,
  column MIN: 1.1
}
```

**Dialect:** mssql
**Sequelize version:** 3.24.4
